optaplanner-examples/src/main/resources/org/optaplanner/examples/tsp/solver/tspSolverConfig.xml

```diff
@@ -10,8 +10,8 @@
   <scoreDirectorFactory>
     <!--<easyScoreCalculatorClass>org.optaplanner.examples.tsp.optional.score.TspEasyScoreCalculator</easyScoreCalculatorClass>-->
     <!--<constraintProviderClass>org.optaplanner.examples.tsp.optional.score.TspConstraintProvider</constraintProviderClass>-->
-    <!--<incrementalScoreCalculatorClass>org.optaplanner.examples.tsp.optional.score.TspIncrementalScoreCalculator</incrementalScoreCalculatorClass>-->
-    <scoreDrl>org/optaplanner/examples/tsp/solver/tspConstraints.drl</scoreDrl>
+    <incrementalScoreCalculatorClass>org.optaplanner.examples.tsp.optional.score.TspIncrementalScoreCalculator</incrementalScoreCalculatorClass>
+    <!-- <scoreDrl>org/optaplanner/examples/tsp/solver/tspConstraints.drl</scoreDrl> -->
```

Comment:
-1 Let's not commit this.

A great way to avoid these uninteded "tmp local changes" is to make a change list in IntelliJ and put that file in that changelist. (I call that change list "dnc" which stands for "Do Not Commit").